hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoUpdateSuite.java

```diff
@@ -132,7 +132,6 @@ public List<HapiSpec> getSpecsInSuite() {
                 updateStakingFieldsWorks());
     }
 
-    @HapiTest
```

Comment:
This was passing as recently as this morning.

Reply:
Okay, let's debug in another PR, to keep this one focused.